pandas/core/algorithms.py

```diff
@@ -743,7 +742,7 @@ def value_counts(
 
     else:
 
-        if is_extension_array_dtype(values) or is_sparse(values):
+        if is_extension_array_dtype(values):
```

Comment:
For everyone's edification: why can we remove this check?

Reply:
is_sparse is a subset of is_extension_array_dtype